.gitignore

```diff
@@ -21,6 +21,7 @@ local.*
 local-*.*
 .env.local
 .env.*.local
+idplist-local.*
```

Comment:
Instead of baking in this, let's try editing Line 20 to be `*local.*` instead to see if that also includes this file pattern, as it's a reasonable assumption that we want to ignore anything that has the word local in it.